src/main/java/dev/socketmods/socketnukes/networking/Network.java

```diff
@@ -0,0 +1,51 @@
+package dev.socketmods.socketnukes.networking;
+
+import dev.socketmods.socketnukes.networking.packet.ExploderConfigChangedPacket;
+import dev.socketmods.socketnukes.networking.packet.ExtendedExplosionPacket;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.network.NetworkDirection;
+import net.minecraftforge.fml.network.NetworkRegistry;
+import net.minecraftforge.fml.network.simple.SimpleChannel;
+
+import static dev.socketmods.socketnukes.SocketNukes.MODID;
+
+/**
+ * Central network handling class.
+ * All of the packets sent for the mod should go through one of the channels here.
+ * All of the packets sent by the mod should go through one of the helper methods here.
+ */
+public class Network {
+    private static final String networkVer = "1";
+
+    public static final SimpleChannel CHANNEL = NetworkRegistry.newSimpleChannel(
+            new ResourceLocation(MODID, "explosions"),
+            () -> networkVer,
+            networkVer::equals,
+            networkVer::equals
+    );
+
+    // Register valid packets, called from SocketNukes, the main class
+    public static void setup() {
```

Comment:
Use a local variable for the packet id and increment? `id++`

This has the downside that we can only append to the end of this method afterwards to not break compatibility